recipes/law/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set name = "law" %}
+{% set version = "0.0.41" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3b6e457bb99da69ddab03e91a80057958ee9477dba7bb6f8823059b2cbce19b3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
```

Comment:
I think this could be a noarch python package

Reply:
Ok, will change that ✔︎.

Reply:
@scopatz  This change resulted in `noarch packages can't have selectors` ...
So does having selectors for dependencies rule out the possibility of making this a noarch python package?

Reply:
Yes, it does. But I don't think you need selectors in this recipe

Reply:
How can I enforce using a requirement below a certain version, depending on the python version? In this case `luigi`, which dropped python 2 support from 3.0 onwards (thus the `>=2.8.2,<3` pin).

Reply:
Hmm yeah, there seems to be a bug in the luigi-feedstock here.  Maybe you do need selectors here right now